opencog/atoms/core/ScopeLink.cc

```diff
@@ -255,9 +255,14 @@ ContentHash ScopeLink::compute_hash() const
 	}
 	fnv1a_hash(hsh, vth);
 
+	const Variables & vars = this->get_variables();
+	for(const auto& pr: vars._glob_intervalmap){
```

Comment:
@noskill you can actually use `_varlist` instead of `get_variables()` (sorry I misguided you.)